app/car.py

```diff
@@ -0,0 +1,21 @@
+from __future__ import annotations
+from math import sqrt
+
+
+class Car:
+    def __init__(self, brand: str, fuel_consumption: float) -> None:
+        self.brand = brand
+        self.fuel_consumption = fuel_consumption
+
+    @classmethod
+    def create_car(cls, car_info: dict) -> Car:
+        return cls(car_info["brand"], car_info["fuel_consumption"])

```

Comment:
The `create_car` method assumes that the dictionary `car_info` will always contain the keys 'brand' and 'fuel_consumption'. It would be safer to add error handling to manage cases where these keys might be missing, such as using `car_info.get('brand')` with a default value or raising an informative exception.